src/navier_stokes/INSVCStaggeredConservativeMassMomentumSSPRKIntegrator.cpp

```diff
@@ -0,0 +1,531 @@
+                w1 = 0.25 * (2.0 + omega);
+                w2 = 0.25 * (dt + 2.0 * d_dt_prev) / sum_dt;
+            }
+            else
+            {
+                w0 = -0.5 * omega, w1 = 1.0 + 0.5 * omega, w2 = 0.0;
+            }
+            break;
+        default:
+            TBOX_ERROR("This statement should not be reached");
+        }
+        // Fill ghost cells for new density and velocity, if needed
+        if (step > 0)
+        {
+            std::vector<InterpolationTransactionComponent> update_transaction_comps(1);
+            update_transaction_comps[0] = InterpolationTransactionComponent(d_rho_scratch_idx,
+                                                                            d_rho_new_idx,
+                                                                            "CONSERVATIVE_LINEAR_REFINE",
+                                                                            false,
+                                                                            "CONSERVATIVE_COARSEN",
+                                                                            d_density_bdry_extrap_type,
+                                                                            false,
+                                                                            d_rho_bc_coefs);
+            d_hier_rho_bdry_fill->resetTransactionComponents(update_transaction_comps);
+            d_hier_rho_bdry_fill->setHomogeneousBc(homogeneous_bc);
+            d_hier_rho_bdry_fill->fillData(eval_time);
+            d_hier_rho_bdry_fill->resetTransactionComponents(d_rho_transaction_comps);
+
+            // Compute an approximation to velocity at eval_time Note, enforce
+            // divergence free condition on all physical boundaries to ensure
+            // boundedness of density update
+            d_hier_sc_data_ops->linearSum(
+                d_V_composite_idx, w0, d_V_old_idx, w1, d_V_current_idx, /*interior_only*/ true);
+            d_hier_sc_data_ops->axpy(d_V_composite_idx, w2, d_V_new_idx, d_V_composite_idx, /*interior_only*/ true);
+            std::vector<InterpolationTransactionComponent> v_update_transaction_comps(1);
+            v_update_transaction_comps[0] = InterpolationTransactionComponent(d_V_scratch_idx,
+                                                                              d_V_composite_idx,
+                                                                              "CONSERVATIVE_LINEAR_REFINE",
+                                                                              false,
+                                                                              "CONSERVATIVE_COARSEN",
+                                                                              d_velocity_bdry_extrap_type,
+                                                                              false,
+                                                                              d_u_bc_coefs);
```

Comment:
```suggestion
            std::vector<InterpolationTransactionComponent> v_update_transaction_comps({{
                                                                              d_V_composite_idx,
                                                                              "CONSERVATIVE_LINEAR_REFINE",
                                                                              false,
                                                                              "CONSERVATIVE_COARSEN",
                                                                              d_velocity_bdry_extrap_type,
                                                                              false,
                                                                              d_u_bc_coefs}});
```
(This would need to be clang-formatted.)